docs/reST/ref/font.rst

```diff
@@ -8,29 +8,31 @@
 
 | :sl:`pygame module for loading and rendering fonts`
 
-The font module allows for rendering TrueType fonts into a new Surface object.
-It accepts any UCS-2 character ('\u0001' to '\uFFFF'). This module is optional
-and requires SDL_ttf as a dependency. You should test that :mod:`pygame.font`
-is available and initialized before attempting to use the module.
-
-Most of the work done with fonts are done by using the actual Font objects. The
-module by itself only has routines to initialize the module and create Font
-objects with ``pygame.font.Font()``.
-
-You can load fonts from the system by using the ``pygame.font.SysFont()``
-function. There are a few other functions to help lookup the system fonts.
-
-Pygame comes with a builtin default font. This can always be accessed by
-passing None as the font name.
-
-To use the :mod:`pygame.freetype` based ``pygame.ftfont`` as
-:mod:`pygame.font` define the environment variable PYGAME_FREETYPE before the
-first import of :mod:`pygame`. Module ``pygame.ftfont`` is a :mod:`pygame.font`
-compatible module that passes all but one of the font module unit tests:
-it does not have the UCS-2 limitation of the SDL_ttf based font module, so
-fails to raise an exception for a code point greater than '\uFFFF'. If
-:mod:`pygame.freetype` is unavailable then the SDL_ttf font module will be
-loaded instead.
+The font module allows for rendering TrueType fonts into Surface objects.
+This module is built on top of the SDL_ttf library, which comes with all
+normal pygame installations.
+
+Most of the work done with fonts are done by using the actual Font objects.
+The module by itself only has routines to support the creation of Font objects
+with :func:`pygame.font.Font`.
+
+You can load fonts from the system by using the :func:`pygame.font.SysFont`
+function. There are a few other functions to help look up the system fonts.
+
+Pygame comes with a builtin default font, freesansbold. This can always be
+accessed by passing ``None`` as the font name.
+
+Before pygame 2.0.3, pygame.font accepts any UCS-2 / UTF-16 character
+('\\u0001' to '\\uFFFF'). After 2.0.3, pygame.font built with SDL_ttf
+2.0.15 accepts any valid UCS-4 / UTF-32 character 
+(like emojis, if the font has them) ('\\U00000001' to '\\UFFFFFFFF')).
```

Comment:
minor doc nitpick here, the largest valid unicode codepoint is `"\U0010FFFF"`, trying to input strings like `"\U00110000"` and above is a `SyntaxError` in python.

Reply:
Thank you, I saw that in testing but didn't know the actual thing to put. Is there a reason python doesn't support `"\U00110000"` to `"\U0010FFFF"` of unicode?

Reply:
According to the [unicode standard](https://en.wikipedia.org/wiki/Unicode), `"\U0010FFFF"` is the limit on unicode (this limit was chosen keeping in mind the max size of UTF-16 surrogate pair encoded characters